src/storage/gcloud-storage/storage.ts

```diff
@@ -0,0 +1,57 @@
+import { Storage } from '@google-cloud/storage';
+import type { Account } from 'account/account.entity';
+import type { AccountService } from 'account/account.service';
+import type { Context } from 'hono';
+import { v4 as uuidv4 } from 'uuid';
+
+export function createStorageHandler(accountService: AccountService) {
+    /**
+     * Handle an upload to GCloud Storage bucket
+     */
+    return async function handleUpload(ctx: Context) {
+        const logger = ctx.get('logger');
+        const storage = new Storage();
+        const bucketName = process.env.GCP_BUCKET_NAME;
+
+        if (!bucketName) {
+            return new Response('Bucket name is not configured', {
```

Comment:
I think this should error at application boot time, not on a request!

Reply:
Done!